components/Projects/ProjectPage/TechList/index.tsx

```diff
@@ -3,17 +3,25 @@ import { jsx } from '@emotion/core'
 import { useTheme } from 'emotion-theming'
 import { styles } from './styles'
 
-const TechList = () => {
+const TechList = ({ data }: Props) => {
   const theme = useTheme()
-
+  console.log(data)
   return (
-    <div className='d-flex align-items-center monospace' css={styles(theme)}>
-      <div className='bg-primary mr-3 bigger'>
-        <i className='material-icons-sharp text-secondary p-2 bigger'>menu</i>
-      </div>
-      VueJs
-    </div>
+    <>
+      {data.map((tech: any, i: number) => (
```

Comment:
using index is an anti-pattern, try using a unique identifier like the name of the tech, or icon, etc